src/test/java/org/jabref/logic/layout/format/AuthorsTest.java

```diff
@@ -123,6 +167,19 @@ public void testMiddleInitial() {
                 a.format("Bruce, Bob Croydon and Charles Kermit von Manson and Jumper, Jolly"));
     }
 
+    /**
+     * Test the FirstInitial method in abbreviation of authors.
+     * setArgument() will pass the String into handleArgument() to set abbreviate.
+     * Increase branch coverage from 58% to 61%.
+     */
+    @Test
+    public void testFirstInitial() {
+        ParamLayoutFormatter a = new Authors();
+        a.setArgument("FirstInitial");
+        assertEquals("B. Bruce, C. von Manson and J. Jumper",
+                a.format("Bruce, Bob Croydon and Charles Kermit von Manson and Jumper, Jolly"));
```

Comment:
Why is it `a.format("Bruce, Bob Croydon and Charles Kermit von Manson and Jumper, Jolly")` instead of `a.format("Bob Croydon Bruce and Charles Kermit von Manson and Jolly Jumper")` like the other tests? Perhaps I missed something ^^

Reply:
I copy that from the other unit test called `testMiddleInitial()` and it uses `a.format("Bruce, Bob Croydon and Charles Kermit von Manson and Jumper, Jolly")`. But I think it will get the same result if I change to `a.format("Bob Croydon Bruce and Charles Kermit von Manson and Jolly Jumper")`. I can try it.

Reply:
Now I get the result. It also can pass the unit test when I change into `a.format("Bob Croydon Bruce and Charles Kermit von Manson and Jolly Jumper")`.